### ISSUE TITLE
Bootstrap template fails with Rails version incompatibility under Rails 6

### ISSUE THREAD

**Issue by dmolesUC** [geoblacklight/geoblacklight#893]
### Descriptive summary

When running the bootstrap template under Rails 6, the application setup fails with a version incompatibility. Workaround (see below) is to install and specify Rails 5 on the command line, so maybe just a documentation issue?

### Expected behavior

New app is generated.

### Actual behavior

Generate fails at the `bundle install from "."` step, with the error:

```txt
Bundler could not find compatible versions for gem "rails":
  In Gemfile:
    rails (~> 6.0.2, >= 6.0.2.2)

    geoblacklight (~> 2.0) was resolved to 2.3.0, which depends on
      rails (~> 5.0)
```

### Steps to reproduce the behavior

In the following environment:

- Ruby version: `ruby 2.6.5p114 (2019-10-01 revision 67812) [x86_64-darwin18]` (via `rvm`)
- macOS version: `10.15.4`
- globally installed Rails version: `6.0.2.2`

Following the [Installation](https://github.com/geoblacklight/geoblacklight/wiki/Installation) instructions on the wiki, execute the following command:

```sh
$ rails new geoblacklight-test -m  \
https://raw.githubusercontent.com/geoblacklight/geoblacklight/master/template.rb
```

### Workaround

```sh
$ gem install rails --version 5.2.4.2
$ rails _5.2.4.2_ new geoblacklight-test -m \
https://raw.githubusercontent.com/geoblacklight/geoblacklight/master/template.rb
```

**Comment by mejackreed** [comment]
This hopefully is resolved with #939 .

**Comment by christinach** [comment]
resolved by https://github.com/geoblacklight/geoblacklight/pull/939
